game.py

```diff
@@ -0,0 +1,4 @@
+from gamefile import *
+
+if __name__ == "game":
+    print("Welecome to the HangMan Game!")
```

Comment:
All what your `main.py` should do using the code you wrote here is to print the "Welcome to the HangMan Game!" message and THAT'S IT - NOTHING ELSE!